Packs/Troubleshoot/Layouts/layoutscontainer-Integration_Troubleshooting.json

```diff
@@ -0,0 +1,404 @@
+     }
+    ],
+    "isVisible": true,
+    "name": "Basic Information",
+    "query": null,
+    "queryType": "",
+    "readOnly": false,
+    "type": ""
+   },
+   {
+    "description": "To use the Troubleshooting Incident, you must have a configured [Demisto REST API integration](https://xsoar.pan.dev/docs/reference/articles/integrations-and-incident-health-check#1-demisto-rest-api-integration)",
+    "fields": [
+     {
+      "fieldId": "incident_troubleshoottype",
+      "isVisible": true
+     },
+     {
+      "fieldId": "incident_instancename",
+      "isVisible": true
+     },
+     {
+      "fieldId": "incident_runwithinsecure",
+      "isVisible": true
+     },
+     {
+      "fieldId": "incident_commandline",
+      "isVisible": true
+     }
+    ],
+    "isVisible": true,
+    "name": "Troubleshooting",
+    "query": null,
+    "queryType": "",
+    "readOnly": false,
+    "type": ""
+   }
+  ]
+ },
+ "group": "incident",
+ "id": "Integration Troubleshooting Layout",
+ "name": "Integration Troubleshooting Layout",
+ "quickView": {
+  "sections": [
+   {
+    "description": "",
+    "fields": [
+     {
+      "fieldId": "incident_type",
+      "isVisible": true
+     },
+     {
+      "fieldId": "incident_severity",
+      "isVisible": true
+     },
+     {
+      "fieldId": "incident_owner",
+      "isVisible": true
+     },
+     {
+      "fieldId": "incident_dbotstatus",
+      "isVisible": true
+     },
+     {
+      "fieldId": "incident_sourcebrand",
+      "isVisible": true
+     },
+     {
+      "fieldId": "incident_sourceinstance",
+      "isVisible": true
+     },
+     {
+      "fieldId": "incident_playbookid",
+      "isVisible": true
+     },
+     {
+      "fieldId": "incident_phase",
+      "isVisible": true
+     },
+     {
+      "fieldId": "incident_roles",
+      "isVisible": true
+     }
+    ],
+    "isVisible": true,
+    "name": "Basic Information",
+    "query": null,
+    "queryType": "",
+    "readOnly": false,
+    "type": ""
+   },
+   {
+    "description": "",
+    "fields": [
+     {
+      "fieldId": "incident_occurred",
+      "isVisible": true
+     },
+     {
+      "fieldId": "incident_dbotcreated",
+      "isVisible": true
+     },
+     {
+      "fieldId": "incident_dbotduedate",
+      "isVisible": true
+     },
+     {
+      "fieldId": "incident_dbotmodified",
+      "isVisible": true
+     },
+     {
+      "fieldId": "incident_dbottotaltime",
+      "isVisible": true
+     }
+    ],
+    "isVisible": true,
+    "name": "Timeline Information",
+    "query": null,
+    "queryType": "",
+    "readOnly": false,
+    "type": ""
+   },
+   {
+    "description": "",
+    "fields": [
+     {
+      "fieldId": "incident_labels",
+      "isVisible": true
+     }
+    ],
+    "isVisible": true,
+    "name": "Labels",
+    "query": null,
+    "queryType": "",
+    "readOnly": true,
+    "type": "labels"
+   }
+  ]
+ },
+ "system": false,
+ "version": -1,
+ "fromVersion": "6.0.0",
+ "description": ""
```

Comment:
please add description here

Reply:
This description is empty in any layoutscontainer that we have in the repo